Lib/test/test_utf8_mode.py

```diff
@@ -12,7 +12,7 @@
 
 MS_WINDOWS = (sys.platform == 'win32')
 POSIX_LOCALES = ('C', 'POSIX')
-
+_vxworks = (sys.platform == "vxworks")
```

Comment:
Please rename the constant to VXWORKS.

Reply:
I have made the requested changes; please review again